airflow/operators/trigger_dagrun.py

```diff
@@ -98,6 +120,7 @@ def __init__(
         poke_interval: int = 60,
         allowed_states: list | None = None,
         failed_states: list | None = None,
+        defer: bool = False,
```

Comment:
I believe the established convention for this is `deferrable`.

Reply:
You're right I mis remembered a conversation I explicitly had about that convention earlier.